runtime/lua/vim/lsp.lua

```diff
@@ -1212,6 +1215,13 @@ function lsp.start_client(config)
     if vim.bo[bufnr].formatexpr == 'v:lua.vim.lsp.formatexpr()' then
       vim.bo[bufnr].formatexpr = nil
     end
+    local keymaps = api.nvim_buf_get_keymap(bufnr, "n")
```

Comment:
You may want to use `  local map = vim.fn.maparg("K", "n", false, true)` here instead which is about 200x faster than getting all keymaps

Reply:
Thanks for pointing out `maparg`, wasn't aware of it.

It's btw. only a 2-4x difference :)

Reply:
In my config, it was about 1ms for getting all of them and 0.006ms with maparg. Profiles it on running the code 100 times with luajit cache cleared. I do have a lot of keymaps though 🙂